arch/arm/core/aarch32/cortex_m/CMakeLists.txt

```diff
@@ -13,11 +13,23 @@ zephyr_library_sources(
   thread_abort.c
   )
 
-zephyr_linker_sources_ifdef(CONFIG_SW_VECTOR_RELAY
+if (CONFIG_SW_VECTOR_RELAY)
+if (CONFIG_CPU_CORTEX_M_HAS_VTOR)
```

Comment:
What is the use case for SW_VECTOR_RELAY if the CPU has VTOR? I thought SW_VECTOR_RELAY was merely a workaround for not having a VTOR.

Reply:
There has been a request to extend this feature for Mainline cortex-m, see #26276